config/locales/en.yml

```diff
@@ -17,15 +17,94 @@ en:
     menu: 'Menu'
     description: 'The free service for schools in England to list teaching roles and for jobseekers to find them.'
     govuk: 'GOV.UK'
-  cookies_banner:
-    aria_label: Cookies banner
-    body: >-
-      We use cookies to collect information about how you use Teaching Vacancies. This helps us to make our service work
-      as well as possible and helps us to improve it in the future.
-    buttons:
-      accept_all: Accept all cookies
-      set_preferences: Set your preferences
-    heading: Cookies on Teaching Vacancies
+
+  cookies:
+    banner:
+      aria:
+        label: Cookies banner
+      body: >-
+        We use cookies to collect information about how you use Teaching Vacancies. This helps us to make our service work
+        as well as possible and helps us to improve it in the future.
+      buttons:
+        accept_all: Accept all cookies
+        set_preferences: Set your preferences
+      heading: Cookies on Teaching Vacancies
+    form:
+      buttons:
+        save_changes: Save changes
+      legend: Do you want to accept Google Analytics cookies?
+      options:
+        consent: Yes, opt-in to Google Analytics cookies
+        no_consent: No, do not use cookies to track my website usage
+    page:
+      body:
+        anonymity: >-
+          We don't allow Google to use or share our analytics data. We anonymise any personally identifiable data,
+          like your IP address or your postcode before we share it with Google.
+        details:
+          intro: 'Google Analytics stores information about:'
+          list:
+            - the pages you visit on Teaching Vacancies
+            - how long you spend on each page
+            - how you got to the site
+            - what you click on while you're visiting the site
+        explanation: >-
+          We use Google Analytics software to collection information about how you use Teaching Vacancies.
```

Comment:
@cpjmcquillan typo here - 'collection' should be collect. To check, is this the final version of the content from @valentine-carter ?